.gitignore

```diff
@@ -0,0 +1,3 @@
+__pycache__
+.env
+node_modules
```

Comment:
Please add the `.vscode` folder to this as well.  You will also need to stop tracking that file.

Learn more about removing tracking here.  https://stackoverflow.com/questions/24290358/remove-a-folder-from-git-tracking